modules/data_release/php/data_release.class.inc

```diff
@@ -51,70 +54,87 @@ class Data_Release extends \NDB_Menu_Filter
             ON (dr.id=drp.data_release_id) 
             JOIN users u ON (u.ID=drp.userid) 
             WHERE u.UserID="
-            . $DB->quote($user->getUsername());
+                . $DB->quote($user->getUsername());
         }
 
-        $this->group_by     = '';
-        $this->order_by     = 'upload_date';
-        $this->headers      = array(
-                               'file_name',
-                               'version',
-                               'upload_date',
-                              );
-        $this->validFilters = array(
-                               'file_name',
-                               'version',
-                               'upload_date',
-                              );
-        $this->formToFilter = array(
-                               'file_name'   => 'file_name',
-                               'version'     => 'version',
-                               'upload_date' => 'upload_date',
-                              );
-
-        $userUpload     = $user->hasPermission("data_release_upload");
-        $userEditAccess = $user->hasPermission("data_release_edit_file_access");
+        $this->group_by = '';
+        $this->order_by = 'uploadDate';
+    }
 
-        $userids = $DB->pselectColWithIndexKey(
-            "SELECT ID, UserID FROM users",
-            array(),
-            "ID"
-        );
+    /**
+     * Create the form for the data release menu page
+     *
+     * @return void
+     **/
+    function setup(): void
```

Comment:
```suggestion
    function setup()
```

I love return types... but this will throw an error since the function signature for `NDB_Menu_Filter::setup()` does not have a return type.